README.md

```diff
@@ -376,8 +376,15 @@ ilg:
   username: test-username
   password: test-password
   debugging: false
+  decode404: true
 ```
 
+### Encode 404
+You can enable decode 404, if you not want client throw an exception when api response is 404.
```

Comment:
You can disable ?

Reply:
non c'est bien l'activation du decodage, j'ai changé le titre en ### Decode 404